addons/l10n_hk/data/account_tax_template_data.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- Account Tags -->
+    <record id="tax_tag_purchase" model="account.account.tag">
+        <field name="name">Tax Received 0%</field>
+        <field name="applicability">taxes</field>
+    </record>
+    <record id="tax_tag_sale" model="account.account.tag">
+        <field name="name">Tax Paid 0%</field>
+        <field name="applicability">taxes</field>
+    </record>
+
+    <!-- account.tax.template -->
+
+    <!-- for purchase -->
+    <record id="tax_purchase_zero" model="account.tax.template">
+        <field name="chart_template_id" ref="l10n_hk_chart_template"/>
+        <field name="name">Tax Received 0%</field>
+        <field name="description">VAT 0%</field>
+        <field name="amount" eval="0"/>
+        <field name="amount_type">percent</field>
+        <field name="price_include" eval="1"/>
+        <field name="type_tax_use">purchase</field>
+        <field name="tag_ids" eval="[(6,0,[ref('tax_tag_purchase')])]"/>
+    </record>
+
+    <!-- for sale -->
+    <record id="tax_sale_zero" model="account.tax.template">
+        <field name="chart_template_id" ref="l10n_hk_chart_template"/>
+        <field name="name">Tax Paid 0%</field>
+        <field name="description">VAT 0%</field>
+        <field name="amount" eval="0"/>
+        <field name="amount_type">percent</field>
+        <field name="price_include" eval="1"/>
+        <field name="type_tax_use">sale</field>
+        <field name="tag_ids" eval="[(6,0,[ref('tax_tag_sale')])]"/>
+    </record>
+
```

Comment:
only 2 taxes? it doesn't seem right...

Reply:
@yostashiro Would there be a case where we can use the VAT report to differentiate the sales/purchase offshore?
Otherwise: these taxes indeed might be nice to have

Reply:
@elicoidal I think that is rather peculiar use of the VAT report if no tax purpose is involved, although I agree that some users might want to be able to get the numbers for domestic/offshore transactions.  I'd suggest changing the names and descriptions to something else in case we keep these records, as it's probably technically not correct to say VAT 0%.

Reply:
I will not fight for it. HK has no VAT so it is logical that we do not create VAT in the localization.
If needed, specific customer can create them.

Reply:
@rgo-odoo : can you please suppress the taxes ? Thanks